### PR TITLE
Sys 2826 extra proxy signed methods tests

### PR DESCRIPTION
**Pull request by micaelffrancoAV**
SYS-2826: Fail test cases for remaining parameters of the proxy signed methods.

### CODE REVIEW

File: pallets/parachain-staking/src/tests/nominate_tests.rs [hunk 1]
```diff
@@ -218,6 +230,75 @@ mod proxy_signed_nominate {
...
+                .with_candidates(vec![(collator_1, 10), (collator_2, 10)])
+                .build()
+                .execute_with(|| {
+                    let amount_to_stake = ParachainStaking::min_total_nominator_stake() * 2u128;
+                    let nonce = ParachainStaking::proxy_nonce(staker.account_id);
+                    let bad_targets = vec![];
+                    let proof = create_proof_for_signed_nominate(
+                        nonce,
+                        &staker,
+                        &vec![collator_1, collator_2],
```
Comment (nahuseyoum) [inline pallets/parachain-staking/src/tests/nominate_tests.rs:290]: :+1:

File: pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs [hunk 2]
```diff
@@ -184,6 +196,44 @@ mod proxy_signed_schedule_revoke_nomination {
...
+                    (staker.relayer, 10000),
+                ])
+                .with_candidates(vec![(collator_1, 10), (collator_2, 10)])
+                .with_nominations(vec![
+                    (staker.account_id, collator_1, 10),
+                    (staker.account_id, collator_2, 10),
+                ])
+                .build()
+                .execute_with(|| {
+                    let bad_nonce = ParachainStaking::proxy_nonce(staker.account_id);
```
Comment (nahuseyoum) [inline pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs:221]:
```suggestion
                    let nonce = ParachainStaking::proxy_nonce(staker.account_id);
```

File: pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs [hunk 3]
```diff
@@ -184,6 +196,44 @@ mod proxy_signed_schedule_revoke_nomination {
...
+                    let proof = create_proof_for_signed_schedule_revoke_nomination(
+                        bad_nonce,
```
Comment (nahuseyoum) [inline pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs:223]:
```suggestion
                        nonce,
```

File: pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs [hunk 4]
```diff
@@ -519,5 +581,55 @@ mod proxy_signed_execute_revoke_all_nomination {
                     );
                 });
         }
+
+        #[test]
+        fn proxy_proof_nominator_is_not_valid() {
+            let collator_1 = to_acc_id(1u64);
+            let collator_2 = to_acc_id(2u64);
+            let bad_nominator = to_acc_id(2000u64);
```
Comment (nahuseyoum) [inline pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs:589]:
```suggestion

```

File: pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs [hunk 5]
```diff
@@ -519,5 +581,55 @@ mod proxy_signed_execute_revoke_all_nomination {
...
+                ])
+                .build()
+                .execute_with(|| {
+                    schedule_leave(staker.clone());
+
+                    // Roll foreward by "Delay" eras to activate leave
+                    roll_to_era_begin((ParachainStaking::delay() + 1u32) as u64);
+
+                    let nonce = ParachainStaking::proxy_nonce(staker.account_id);
+
```
Comment (nahuseyoum) [inline pallets/parachain-staking/src/tests/schedule_revoke_nomination_tests.rs:611]:
```suggestion
    let bad_nominator = to_acc_id(2000u64);
```

**Review by nahuseyoum — APPROVED** [review]

**Review by alexpArtos — APPROVED** [review]

File: pallets/parachain-staking/src/tests/bond_extra_tests.rs [hunk 6]
```diff
@@ -463,6 +516,43 @@ mod proxy_signed_candidate_bond_extra {
                 });
         }
 
+        #[test]
+        fn proxy_proof_extra_amount_is_not_valid() {
```
[the remark this answers is not in the document]
Comment (alexpArtos) [reply pallets/parachain-staking/src/tests/bond_extra_tests.rs:520]: Unless we do mean to test a valid signature is required in proxy tests. Then I think we may have to test each field in the signature.

**Review by fluorostani — APPROVED** [review]

**Pull request merged**